src/sentry/utils/relocation.py

```diff
@@ -36,6 +42,205 @@ class OrderedTask(Enum):
 RELOCATION_BLOB_SIZE = int((2**31) / 32)
 
 
+# Create the relevant directories: a `/workspace/in` directory containing the inputs that will
+# be imported, a `/workspace/out` directory for exports that will be generated, and
+# `/workspace/findings` for findings.
+#
+# TODO(getsentry/team-ospo#203): Make `get-self-hosted-repo` pull a pinned version, not
+# mainline.
+#
+# TODO(getsentry/team-ospo#203): Use script in self-hosted to completely flush db instead of
+# using truncation tables.
+CLOUDBUILD_YAML_TEMPLATE = Template(
+    """
+steps:
+
+  ##############################
+  ### Setup steps
+  ##############################
+
+  - name: "gcr.io/cloud-builders/gsutil"
+    id: copy-inputs-being-validated
+    waitFor: ["-"]
+    args: ["cp", "-r", "$bucket_root/relocations/runs/$uuid/in", "."]
+    timeout: 600s
+
+
+  - name: "gcr.io/cloud-builders/docker"
+    id: create-working-dirs
+    waitFor: ["-"]
+    entrypoint: "bash"
+    args:
+      - "-e"
+      - "-c"
+      - |
+        mkdir /workspace/out && chmod 777 /workspace/out
```

Comment:
wondering if chmod 755 or 775 can be used here just out of practice of not giving everyone full permissions

Reply:
Tested it out - needs to be 777 to give the compare script running in docker proper access.